js/capture.js

```diff
@@ -19,13 +27,61 @@ const capture = {
       });
   },
 
+  // In the parsed whitelist JSON object (from disconnect-entitylist.json):
+  //   - 'properties' keys are first parties
+  //   - 'resources' keys are third parties
+  reformatList(whiteList) {
```

Comment:
This is basically just how I code:
![giphy](https://user-images.githubusercontent.com/71928/28141465-473b182a-6722-11e7-9a1c-bd364aabf0b8.gif)